### PR TITLE
Custom field views for specific action

### PR DESCRIPTION
**Pull request by jimmysp**
Hi, first of all, this is my first PR, hope I got it right. 
I would like to improve the custom field views, having options for each separate action. I also made an edit in the documentation section, the custom views can be set like this:
{
    "view_browse": "my_browse_view",
    "view_read": "my_read_view",
    "view_add": "my_add_view",
    "view_edit": "my_edit_view",
    "view_order": "my_order_view"
}

### CODE REVIEW

File: docs/bread/formfields/introduction.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Formfields
 
-Formfields are the hearth of Voyagers BREAD-system.
```
Comment (emptynick) [inline docs/bread/formfields/introduction.md]: This tells the markdown parser to add a line-break here which we want.
Please revert those changes (same for the other lines below)

**Review by emptynick — DISMISSED** [review]

**Comment by jimmysp** [comment]
Ok, I restored the whitespaces with a new commit.

**Comment by emptynick** [comment]
I think for consistency it would be great to pass `$view` as well.
This way you could also have the same view for browse and read for example, which are very similar from code perspective.

**Comment by jimmysp** [comment]
Indeed, it could be useful to use the same view for similar actions... I pushed a new commit for this.

**Review by emptynick — APPROVED** [review]

**Pull request merged**